src/Components/People.js

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+import React, { Component } from "react";
+
+export default class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      input: "",
+      people: [],
+      person: {},
+      notFound: false,
+    };
+  }
+  handleChange = (e) => {
+    this.setState({
+      input: e.target.value,
+    });
+  };
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { input, people } = this.state;
+    people.forEach((person) => {
+      if (person.name === input) {
+        console.log(person);
+        this.setState({person})
+      } else{
+          console.log("dub")
+          this.setState({notFound:true})
+      }
```

Comment:
This might cause issues. What If I search for 'San' and San is the second person in the array of people?